modules/combined/tests/creep_plasticity_combined/combined_creep_plasticity.i

```diff
@@ -9,7 +9,6 @@
 #   further yielding during unloading. The initial yield stress (sigma_0) is prescribed
 #   as 20 with a plastic strain hardening of 100. The mesh is a 1x1x1 cube with symmetry
 #   boundary conditions on three planes to provide a uniaxial stress field.
-#   The temperature is held constant at 1000.
```

Comment:
A  couple of questions about this test:
1. It looks like you removed the heat transfer part of it.  Could these tests be moved to tensor_mechanics now? It's good to move any tests that we can out of modules/combined/tests.
2. Did the temperature not affect the results before?